src/idea_evaluation.md

```diff
@@ -0,0 +1,50 @@
+# Evaluation of the idea
```

Comment:
What is the "idea" referenced here? This page seems to describe different types or categories of platforms. It would be nice to explain how this relates to LinuxBoot in the text. How does this help someone coming to learn about using or implementing LinuxBoot?

Reply:
Good point, I'll rework that and elaborate.